frontend/javascripts/test/fixtures/dummy_organization.ts

```diff
@@ -0,0 +1,19 @@
+import { APIOrganization } from "types/api_flow_types";
+import { PricingPlanEnum } from "admin/organization/pricing_plan_utils";
+
+const dummyOrga: APIOrganization = {
+  id: "dummy_orga_id",
+  name: "dummy_orga",
+  additionalInformation: "more information",
+  displayName: "Dummy Orga",
+  pricingPlan: PricingPlanEnum.Basic,
+  enableAutoVerify: true,
+  newUserMailingList: "dummy@orga.com",
```

Comment:
```suggestion
  newUserMailingList: "dummy@example.com",
```

I know it's not used but this is [best practice](https://www.iana.org/domains/reserved).

Reply:
ah, good to know, thanks :pray: